### DISCUSSION TITLE
Support git lfs

### DISCUSSION THREAD

**Discussion by wrvsrx** [damionx7/Zettel-Notes-Documentation#60] [Feature Requests]
My note repo has a lot of images in it, so I use git lfs to reduce checkout size. Would you consider support git lfs?

**Comment by damionx7** [comment]
Have you tried git synchronization in Zettel Notes. Does it throw any error with existing implementation ?

**Comment by wrvsrx** [comment]
No. It simply doesn't sync images to the device.

**Comment by damionx7** [comment]
I will try to solve this issue.

**Comment by damionx7** [comment]
[Adam]: Git repos can get big over time with lots of 20 minute audio recordings and drawings when you frequently add and delete. Adding support for Git LFS would help mitigate this issue.

**Comment by sherifkhalifa** [comment]
any progress or possibility of adding LFS in a future update??

**Reply by damionx7** [reply]
I will have to explore git library integration via ndk.
